api/src/zero/module/wgs.clj

```diff
@@ -112,23 +113,26 @@
 
 (defn make-inputs
   "Return inputs for reprocessing IN-GS into OUT-GS in ENVIRONMENT."
-  [environment out-gs in-gs]
+  [environment out-gs in-gs sample]
   (let [[input-key base _] (all/bam-or-cram? in-gs)
         leaf (last (str/split base #"/"))
         [_ out-dir] (gcs/parse-gs-url (util/unsuffix base leaf))
+        ref-prefix (get sample :reference_fasta_prefix)
         inputs (-> (zipmap [:base_file_name :final_gvcf_base_name :sample_name]
                            (repeat leaf))
                    (assoc input-key in-gs)
                    (assoc :destination_cloud_path (str out-gs out-dir))
-                   (assoc :references references)
+                   (assoc :references (make-references ref-prefix))
                    #_(merge per-sample) ;; Uncomment to enable fingerprinting
                    (merge cram-ref)
                    (merge (genome-inputs environment)
                           hack-task-level-values))
-        {:keys [destination_cloud_path final_gvcf_base_name]} inputs
+        merged-inputs (merge inputs sample)
+        {:keys [destination_cloud_path final_gvcf_base_name]} merged-inputs
         output (str destination_cloud_path final_gvcf_base_name ".cram")]
     (all/throw-when-output-exists-already! output)
-    (util/prefix-keys inputs :ExternalWholeGenomeReprocessing)))
+    (util/prefix-keys merged-inputs :ExternalWholeGenomeReprocessing)
+    ))
```

Comment:
```suggestion
    (util/prefix-keys merged-inputs :ExternalWholeGenomeReprocessing)))
```
I don't really have a personal preference here, a lot of our code avoids dangling parens so I figured I'd suggest the edit--doesn't matter to me.